iterations/3.md

```diff
@@ -12,6 +12,6 @@ You may need to revise parts of the [Design](/iterations/1#design), and even par
 Submit the code base for of your application in the root directory of the `master` branch of your group’s repository at `https://github.com/jhu-oose/{{site.course}}-group-<identifier>`.
 </small>
 
-You must implement the core _distinguishing_ features of your application and demonstrate them on the [Midterm Presentation](/group-projects#presentations) by the end of the iteration. The application doesn’t have to be usable yet, but it must be working well enough to show the core concepts in action. What this means depends a lot on the nature of your project, so you must work with your advisor early in the iteration to establish the goals.
+You must implement the core _distinguishing_ features of your application and demonstrate them on the [Midterm Presentation](/group-projects#presentations) by the end of the iteration. The application doesn’t have to be usable yet, but it must be working well enough to show the core concepts in action. The specifics of what this entails depends a lot on the nature of your project, so you must work with your advisor early in the iteration to establish the goals.
```

Comment:
Lovely, please propagate to the next iterations pages, which contain a similar message.

Reply:
Done